backend/internal/domain/helpers/tokens_generator.go

```diff
@@ -0,0 +1,62 @@
+package helpers
+
+import (
+	"os"
+	"shodo/models"
+	"time"
+
+	"github.com/golang-jwt/jwt"
+)
+
+const (
+	AccessTokenLifeTime  = 3600
+	RefreshTokenLifeTime = 86400
+)
+
+var Secret = os.Getenv("JWT_SECRET")
+
+func GenerateTokens(userId string) (models.AuthTokens, error) {
```

Comment:
NIT: Common words like id, url, etc should follow the same casing
So you can use either: id or ID, but not Id
Same goes for URL or url, not Url

This is a nit but usually most style guides are following this convention